### PR TITLE
Publish - July 13th, 2020

### PR DESCRIPTION
**Pull request by afebbraro**
## What does this PR do?
This is our Summer release work Please see the release notes for all changes.

### CODE REVIEW

File: html/components/_tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  }
+
+  &:before {
+    position: absolute;
+    bottom: -0.5rem;
+    left: 1rem;
+    width: $sprk-tooltip-caret-width;
+    height: $sprk-tooltip-caret-width;
+    background-color: $sprk-tooltip-caret-background-color;
+    transform: rotateY(0deg) rotate(45deg);
```
Comment (afebbraro) [inline html/components/_tooltip.scss:62]: Lets add vendor prefixes for this transform.

File: html/components/_tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,134 @@
+////
+/// @group tooltip
+////
+
+.sprk-c-Tooltip__container {
+  position: relative;
+  display: inline-flex;
```
Comment (afebbraro) [inline html/components/_tooltip.scss:7]: Does the inline-flex on tooltip work in IE11 okay without the vendor prefixes? Think we need those here.

File: html/settings/_settings.scss [hunk 3]
```diff
@@ -1530,7 +1648,45 @@ $sprk-toggle-trigger-font-weight-small: normal !default;
...
+/// The background color for the tooltip.
+$sprk-tooltip-background-color: $sprk-black !default;
+/// The font color for the tooltip.
+$sprk-tooltip-font-color: $sprk-white !default;
+/// The text align for the tooltip.
+$sprk-tooltip-text-align: left !default;
+/// The font weight for the tooltip
+$sprk-tooltip-font-weight: 300 !default;
+/// The padding for the tooltip.
+$sprk-tooltip-padding: $sprk-space-inset-short-m;
```
Comment (afebbraro) [inline html/settings/_settings.scss:1673]: Missing the !default here.

File: react/src/base/inputs/SprkCheckbox/SprkCheckboxGroup/SprkCheckboxGroup.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+    if (element.type.name === 'SprkHelperText') {
+      hasHelperText = true;
+      helperId = element.props.id || uniqueId('sprk-helper-text-');
+      return React.cloneElement(element, { id: helperId });
+    }
+    return element;
+  });
+
+  // For each element, if it has granChildren, add ariaDescribedBY to those
```
Comment (afebbraro) [inline react/src/base/inputs/SprkCheckbox/SprkCheckboxGroup/SprkCheckboxGroup.js:39]: is this supposed to be granChildren?

File: react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    checkboxAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    disabled,
```
Comment (afebbraro) [inline react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js:17]: Talked with RV, this should be isDisabled to match our naming conventions. He's gonna make a PR for it.

File: react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    checkboxAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    isDisabled,
+    onChangeFunc,
```
Comment (afebbraro) [inline react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js:18]: We recently changed this in other inputs to be onChange instead. I messaged @RVMendoza  about updating this one. Leaving a comment to remember!

File: react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    radioAdditionalClasses,
+    labelAdditionalClasses,
+    name,
+    value,
+    isDisabled,
+    onChangeFunc,
```
Comment (afebbraro) [inline react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js:18]: This needs to be updated to onChange per our recent change we made to other inputs. Messaged @RVMendoza  on this!

File: react/src/base/inputs/SprkRadio.stories.js [hunk 8]
```diff
@@ -0,0 +1,536 @@
...
+                Radio Item 4
+              </SprkRadioItem>
+            </SprkStackItem>
+          </SprkStack>
+        </SprkStackItem>
+        <SprkStackItem>
+          <SprkStack
+            splitAt="small"
+            itemSpacing="medium"
+            additionalClasses="sprk-o-Stack--center-row "
```
Comment (afebbraro) [inline react/src/base/inputs/SprkRadio.stories.js:358]:
```suggestion
            additionalClasses="sprk-o-Stack--center-row"
```

File: src/pages/using-spark/components/tooltips.mdx [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+<SprkDivider
+ element="span"
+ additionalClasses="sprk-u-Measure"
+></SprkDivider>
+
+## Anatomy
+
+- Content within tooltips must be text-based with a maximum character count of 140.
+- The maximum width of tooltips must be 290 px.
```
Comment (afebbraro) [inline src/pages/using-spark/components/tooltips.mdx:44]:
```suggestion
- The maximum width of tooltips must be 290px.
```

File: src/pages/using-spark/components/input.mdx [hunk 10]
```diff
@@ -144,130 +94,148 @@ than one option from a list.
...
+#### Anatomy
+- A question related to the group of options.
+- A list of two to six options, each with a concise label, all the same size.
 
 <ComponentPreview
-  componentName="input-radio--default-story"
-  hasReact
-  hasAngular
+  componentName="input-huge-checkbox--default-story"
   hasHTML
```
Comment (afebbraro) [inline src/pages/using-spark/components/input.mdx:121]: Need to add hasAngular here.

File: html/components/_tooltip.scss (hunk 1 above)
Comment (bobwise) [reply html/components/_tooltip.scss:64]: done

File: html/components/_tooltip.scss (hunk 2 above)
Comment (bobwise) [reply html/components/_tooltip.scss:7]: I tested this again in IE 11 and we're good

File: react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js (hunk 6 above)
Comment (bobwise) [reply react/src/base/inputs/SprkCheckbox/SprkCheckboxItem/SprkCheckboxItem.js:18]: fixed

File: react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js (hunk 7 above)
Comment (bobwise) [reply react/src/base/inputs/SprkRadio/SprkRadioItem/SprkRadioItem.js:18]: fixed

File: src/pages/using-spark/components/input.mdx (hunk 10 above)
Comment (bobwise) [reply src/pages/using-spark/components/input.mdx:121]: fixed

**Pull request merged**